ocs/defaults.py

```diff
@@ -21,69 +20,51 @@
 ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
 KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
 API_VERSION = "v1"
+CEPHFILESYSTEM_NAME = 'ocsci-cephfs'
```

Comment:
Currently, our [CephFilesystem.yaml](https://github.com/red-hat-storage/ocs-ci/blob/master/templates/CSI/cephfs/CephFileSystem.yaml) is a mofdified version of [filesystem.yaml from rook repo](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/filesystem.yaml). Hence we have already changed followign two params: 

 name: myfs
  namespace: rook-ceph

If we plan to use the exact copy of yaml from rook repo in the future, should we have the default CephFilesystem name same as the rook repo and the yaml taken directly from rook repo(similar to templates/CSI/ approach?

Please suggest ?

Reply:
@nehaberry are you suggesting the name used here is different from what we are using in deployment?